### DISCUSSION TITLE
I can’t see through the water blocks in my schematic when I am inside of it

### DISCUSSION THREAD

**Discussion by tamimd67** [maruohon/litematica#810] [Q&A] [answered]
In normal mode, if I am outside my schematic, I can see running water and clouded outlines of coral, fish, etc (1). However, when I step inside the schematic, all I see is a lot of bright blue (2). I tried turning “renderBlocksAsTranslucent” to True and “renderCollidingSchematicBlocks” as True. This improved things from outside of the schematic (3), but not inside (4). Is there a way to still see where the water blocks are, but not be lost in a sea of blue when you are inside the water in the schematic?
![Screenshot 2024-01-24 184101](https://github.com/maruohon/litematica/assets/39843111/51184bc6-0007-4982-97e8-6ca51fc4f0a9)

**Comment by maruohon** [comment]
Currently, probably not... unless the Visuals -> `overlayReducedInnerSides` option already does a good enough job of removing the overlay quads from between same color overlay blocks. I don't remember what the state of that option is, but I know it's still buggy and unfinished.

**Reply by tamimd67** [reply, marked as answer]
That didn't do it, but it sent me in the right direction. I set schematicOverlayEnableOutlines to False, and schematicOverlayEnableSides to False.  Now it looks like this - significant improvement in visibility. Thanks!
![2024-01-25_18 50 03](https://github.com/maruohon/litematica/assets/39843111/465a9a95-d17e-4a86-a43f-12719d969e55)
![2024-01-25_18 50 16](https://github.com/maruohon/litematica/assets/39843111/71df5171-e0c7-4354-b1a7-b05966f84688)

**Comment by maruohon** [comment]
If you don't need the colored overlay at all, then use the option Visuals -> `enableSchematicOverlay` to completely disable it (or the hotkey `toggleOverlayRendering` to quickly toggle it on/off - I like to use `M + O` for that, for **O**verlay).

**Comment by tamimd67** [comment]
Thank you so much!

On Fri, Jan 26, 2024 at 1:08 PM Matti Ruohonen ***@***.***>
wrote:

> If you don't need the colored overlay at all, then use the option Visuals
> -> enableSchematicOverlay to completely disable it (or the hotkey
> toggleOverlayRendering to quickly toggle it on/off - I like to use M + O
> for that, for *O*verlay).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/maruohon/litematica/discussions/810#discussioncomment-8260819>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJP7KJ2LP3IXQQXF6VMATETYQP5JNAVCNFSM6AAAAABCLJLUAKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DENRQHAYTS>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tami Duvall
